benchmark/metrics/cputime.py

```diff
@@ -0,0 +1,216 @@
+# Copyright 2015 ClusterHQ Inc.  See LICENSE file for details.
+"""
+CPU time metric for the control service benchmarks.
+"""
+
+from zope.interface import implementer
+
+from twisted.protocols.basic import LineOnlyReceiver
+
+import eliot
+
+from flocker.common import gather_deferreds
+from flocker.common.runner import run_ssh
+
+from benchmark._interfaces import IMetric
+
+_FLOCKER_PROCESSES = {
+    u'flocker-control',
+    u'flocker-dataset-agent',
+    u'flocker-container-agent',
+    u'flocker-docker-plugin',
+}
+
+
+_GET_CPUTIME_COMMAND = [
+    # Use system ps to collect the information
+    b'ps',
+    # Output the command name (truncated) and the cputime of the process.
+    # `=` provides a header.  Making all the headers blank prevents the header
+    # line from being written.
+    b'-o',
+    b'comm=,cputime=',
+    # Output lines for processes with names matching the following (values
+    # supplied by invoker)
+    b'-C',
+]
+
```

Comment:
The version of `ps` on OS X has different switches compared to the version of `ps` on Linux. As a result, tests which use this command will fail to run on OS X. Given that the tests are designed to run on systems that can run Flocker (CentOS 7, Ubuntu 14.04), should we disable these tests on unsupported systems?